extensions/bombman/standalone.mjs

```diff
@@ -334,8 +334,8 @@ class Standalone {
     } else { // default
       const player = this.helper.gameState.getPlayer(playerID);
       target = player.mapCoord.copy();
-      target.x = 10;
-      target.y = 5;
+      target.x = 1;
+      target.y = 1;
```

Comment:
Consider using spawn point for this:
https://github.com/canyugs/hitcon-online/blob/main/services/gateway/gateway-service.mjs#L322